.travis.yml

```diff
@@ -49,18 +52,26 @@ install:
   - pip install -r requirements.txt
   - pip install .
   - pip install coveralls
+  # install vaxrank and its dependent data
+  - pip install --upgrade -r $PWD/vaxrank/requirements.txt
+  - pyensembl install --release 87 --species human
```

Comment:
@tavinathanson @julia326 Thoughts on how to shrink the footprint of Ensembl release installs? I was thinking about adding a pyensembl option to get rid of noncoding transcripts